src/app/PersistableValues.spec.ts

```
@@ -0,0 +1,17 @@
+import { migrations } from './PersistableValues';
+
+describe('PersistableValues migrations', () => {
+  it('adds telephony shortcut config without losing a persisted telephony server', () => {
+    const before = ({
```

ESLint

[error] Delete `(`

(prettier/prettier)